src/navigators/createMaterialTopTabNavigator.js

```diff
@@ -17,16 +18,35 @@ type Props = InjectedProps & {
   tabBarPosition?: 'top' | 'bottom',
   tabBarComponent?: React.ComponentType<*>,
   tabBarOptions?: TabBarOptions,
+  lazyOnSwipe: boolean,
```

Comment:
Don't think this should be exposed. Setting this to `false` results in a bad user experience and exposing it makes it easier for devs to provide a bad UX. Devs can already acheive this with focus listeners if they really want to provide bad UX.

Reply:
It can be useful in case we don't want to load scenes on swipe, like Facebook does.

But, I'm agree with you that it can be done with focus listeners and we may not want to expose props for bad UX.